cluster/examples/kubernetes-bridge-marker.yml

```diff
@@ -0,0 +1,78 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: bridge-marker
+  namespace: kube-system
+  labels:
+    tier: node
+    app: bridge-marker
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: bridge-marker
+    spec:
+      serviceAccountName: bridge-marker
+      hostNetwork: true
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: bridge-marker
+        image: registry:5000/bridge-marker:latest
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          privileged: true
+        args:
+          - -node-name
+          - $(NODE_NAME)
+        volumeMounts:
+          - name: bridge-sbin
+            mountPath: /host/sbin    
+        env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+      volumes:
+        - name: bridge-sbin
```

Comment:
why we need this?

Reply:
We are using /sbin/ip of the node in the marker

Reply:
can we just install the ip command inside the container?

Reply:
Done